UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Helpers/FileShareService.cs

```diff
@@ -134,6 +135,7 @@ public async Task<SearchBatchResponse> GetBatchInfoBasedOnProducts(List<Products
             } while (httpResponse.IsSuccessStatusCode && internalSearchBatchResponse.Entries.Count != 0 && internalSearchBatchResponse.Entries.Count < prodCount && !string.IsNullOrWhiteSpace(uri));
             internalSearchBatchResponse.QueryCount = queryCount;
             CheckProductsExistsInFileShareService(products, correlationId, batchId, internalSearchBatchResponse, internalNotFoundProducts, prodCount);
+            logger.LogInformation(EventIds.FileShareServicePreparingToQueryENCFilesCompleted.ToEventId(), "Completed file share service request for BatchId:{batchId} based on {productDetails} and _X-Correlation-ID:{CorrelationId}", batchId, productWithAttributes.Item2, correlationId);
```

Comment:
Please can we have log message something like:

```suggestion
            logger.LogInformation(EventIds.FileShareServicePreparingToQueryENCFilesCompleted.ToEventId(), "Completed file share service search request for {productDetails}. ESS BatchId:{batchId} and _X-Correlation-ID:{CorrelationId}",  productWithAttributes.Item2, batchId, correlationId);
```

Reply:
Yes updated message as per suggested, Thanks!